CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -1037,7 +1042,15 @@ void IRenderWebBrowser.OnPopupShow(bool isOpen)
         /// <param name="isOpen">if set to <c>true</c> [is open].</param>
         protected virtual void OnPopupShow(bool isOpen)
         {
-            UiThreadRunAsync(() => { popupImage.Visibility = isOpen ? Visibility.Visible : Visibility.Hidden; });
+            UiThreadRunAsync(() =>
+            {
+                popupImage.Visibility = isOpen ? Visibility.Visible : Visibility.Hidden;
+
+                if (!isOpen)
+                {
+                    mouseAdjustor.Reset();
```

Comment:
If you rename `Reset` to `OnPopupShow` and always call it passing in `isOpen` then we can track if the popup is open internally.

Reply:
Done.